website/workshops.html

```diff
@@ -0,0 +1,85 @@
+<!--
+ Copyright 2023 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<div id="workshops"></div>
+
+<script>
+  document.addEventListener('DOMContentLoaded', () => {
+    const items = [
+      {
+        date: new Date('2023-06-27'),
+        location: 'Berlin',
+        link: '<a href="https://gdg.community.dev/events/details/google-gdg-berlin-golang-presents-service-weaver-workshop/">Register</a>',
+      },
+      {
+        date: new Date('2023-06-30'),
+        location: 'Zurich',
+        link: '<a href="https://gdg.community.dev/events/details/google-gdg-zurich-presents-service-weaver-workshop/">Register</a>',
+      },
+    ];
+
+    const workshops = document.getElementById('workshops');
+    const block = document.createElement('div');
+    block.classList.add('news-block');
+    block.innerHTML = `
+      <div class="news-content">
+        <div style="color:darkblue"><p>Please reach out to us if you're interested in hosting a Service Weaver workshop in your city or institution!</p></div>
```

Comment:
Maybe add a sentence like "We're hosting a number of workshops where you can get hands-on experience writing Service Weaver apps. Use the links below to sign up."?

Reply:
Good idea. Added.